src/content/plugins-tutorials/home-indicator.md

```diff
@@ -59,6 +59,6 @@ getWindow().setDecorFitsSystemWindows(false);
 
 ## Credits
 
-This plugin was created originally for [Kick.com](https://kick.com) by [Capgo](https://capgo.app)
+This plugin was created originally for [Kick.com](https://g.co/kgs/6Nvu3n) by [Capgo](https://capgo.app)
```

Comment:
i'm not sure why this modif ?

Reply:
Yeah, this shouldn't modify. Thanks for your review.